src/platform/lumin-runtime/platform-information.js

```diff
@@ -0,0 +1,11 @@
+class PlatformInformation {
+    platformName() {
```

Comment:
Can we avoid the duplication of the word `platform` from `platformName` since the class name already identifies that this object represents `PlatformInformation` ? Unless, there is a plan to include additional information which is not specifically related to the platform.